src/Microsoft.Extensions.Configuration.AzureAppConfiguration/Extensions/TimeSpanExtensions.cs

```diff
@@ -144,7 +144,7 @@ private static TimeSpan Jitter(this TimeSpan timeSpan, double ratio)
 
             var rand = new Random();
 
-            double jitter = ratio * (rand.NextDouble() * 2 - 1);
+            double jitter = ratio * ((rand.NextDouble() * 2) - 1);
```

Comment:
`rand.NextDouble() * 2 - 1` looks good enough, the brackets looks unnecessary in `(a*b) -c` and won't cause ambiguity I believe.

So what's the related rule, is it too strict?

Reply:
There are these options:

Now the rule is
dotnet_style_parentheses_in_arithmetic_binary_operators = always_for_clarity
dotnet_style_parentheses_in_relational_binary_operators = always_for_clarity
dotnet_style_parentheses_in_other_binary_operators = always_for_clarity
dotnet_style_parentheses_in_other_operators = always_for_clarity

ref: https://learn.microsoft.com/en-us/dotnet/fundamentals/code-analysis/style-rules/ide0047-ide0048

How about this?
dotnet_style_parentheses_in_arithmetic_binary_operators = never_if_unnecessary
dotnet_style_parentheses_in_relational_binary_operators = always_for_clarity
dotnet_style_parentheses_in_other_binary_operators = always_for_clarity
dotnet_style_parentheses_in_other_operators = never_if_unnecessary

Reply:
By changing it to "never_if_unnecessary", does it means `(a*b) -c` gets a warning? 
Can we simply remove this rule without checking it? My expectation is, no matter I write `(a * b) - c` or `a * b - c`. Because I think this case doesn't really matter much. I would expect to have "critical" rules only.

Reply:
> By changing it to "never_if_unnecessary", does it means (a*b) -c gets a warning?

Yes.

> Can we simply remove this rule without checking it? 

Yes. I can suppress this rule.